Patches/The GiantRace/ThingDefs_Races/AlienRace_Giantlike.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+  <Operation Class="PatchOperationSequence">
+	<operations>
+	  <li Class="PatchOperationFindMod">
+			
+		<mods><li>The GiantRace</li></mods>
```

Comment:
Giants' bulk/weight carrying capacity is fairly low relative to their size, and so may need increased manually.

Reply:
Mechanics wise, it's actually matched exactly with their size, since they're considered bodysize 3.5, or just below an elephant.
I'll increase the bulk a bit though to represent better carrying ability than a conventional animal though. The mass is, imo, fine as is.